src/app/docs/description-list/description-list-docs.component.html

```diff
@@ -0,0 +1,90 @@
+<sky-docs-demo-page
+  moduleName="SkyDescriptionListModule"
+  moduleSourceCodePath="src/app/public/modules/description-list/"
+  pageTitle="Description list"
+>
+  <sky-docs-demo-page-summary>
+    Description lists display lists of terms and pairs each term with a description.
```

Comment:
```suggestion
    Description lists display lists of terms and then pair each of the terms with a description.
```
I think this is my bad on the wording; it sounded good when I suggested it before, but it reads pretty awkwardly. I think these tweaks should fix the problematic wording.